secured/.env

```diff
@@ -0,0 +1,4 @@
+DB_HOST=localhost
```

Comment:
.env usuń z repozytorium, zrób commit, a potem dodaj do .gitignore i znowu zacommituj. Wtedy jak utworzysz plik .env to git nie będzie brał go pod uwagę